.github/dependabot.yml

```diff
@@ -0,0 +1,23 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: "weekly"
+    groups:
+      actions:
+        patterns:
+          - "*"
+  - package-ecosystem: cargo
+    directory: /
+    schedule:
+      interval: "weekly"
+    groups:
+      cargo:
+        patterns:
+          - "*"
```

Comment:
I'm not sure if this is worth enabling. The latest dependabot changes have been troublesome.

- cargo workspace support is awaiting a fix: https://github.com/dependabot/dependabot-core/pull/13359
- egui seems to have breaking changes in every minor release. Dependabot will create a branch that bumps the version(s) specified in manifests (and the lock file). But it will not address the frequently needed migrations between minor versions of egui.

If this works then it works. If not, you can disable dependabot for the `cargo` ecosystem by commenting out this yaml list item.

Not unsurprisingly, I don't foresee any problems with the automatic updates for `github-actions` ecosystem.